src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestSuite.scala

```diff
@@ -32,6 +32,7 @@ class AgoraTestSuite extends Suites(
   new CompatibleConfigurationIntegrationSpec,
   new Ga4ghServiceSpec,
   new Ga4ghModelTest,
+  new SwaggerRoutesSpec,
```

Comment:
while investigating the trailing slashes in swagger routes, I wrote some tests (see below)